src/resources.json

```diff
@@ -194,6 +194,99 @@
         "lists": [ "organizations", "fundraising-teams", "fundraising-pages" ],
         "path": "/members"
     },
+    "MergeTags": {
+        "path": "/merge-tags",
+        "basic": [ "retrieve" ],
+         "custom": {
+            "methods": {
+                "tags": {
+                    "method": "GET",
+                    "path": ""
+                }
+            }
+        }
+    },
+    "MessageAttachments": {
+        "path": "/message-attachments",
+        "basic": [ "retrieve" ],
+        "custom": {
+            "methods": {
+                "attachments": {
+                    "method": "GET",
+                    "path": ""
```

Comment:
I think this will do the same thing as the 'retrieve' for /message-attachments as it's currently written. Should the path be /{id}/attachments?